README.md

```diff
@@ -175,6 +175,7 @@ This provider exposes quite a few provider-specific configuration options:
 * `zone` - The zone name where the instance will be created.
 * `can_ip_forward` - Boolean whether to enable IP Forwarding.
 * `external_ip` - The external IP address to use.
+* `service_accounts` - An array of OAuth2 account scopes for the services the instance will have access to, for example: `['https://www.googleapis.com/auth/devstorage.full_control']`
```

Comment:
What do you think about having vagrant support the `aliases` that are defined by `gcloud compute`?  That would make vagrant more consistent with the command-line utility. See `gcloud compute instances create --help` and look for `scopes`.

```
          Alias              URI
          bigquery           https://www.googleapis.com/auth/bigquery
          cloud-platform     https://www.googleapis.com/auth/cloud-platform
          compute-ro         https://www.googleapis.com/auth/compute.readonly
          compute-rw         https://www.googleapis.com/auth/compute
          computeaccounts-ro https://www.googleapis.com/auth/computeaccounts.readonly
          computeaccounts-rw https://www.googleapis.com/auth/computeaccounts
          datastore          https://www.googleapis.com/auth/datastore
          logging-write      https://www.googleapis.com/auth/logging.write
          monitoring         https://www.googleapis.com/auth/monitoring
          sql                https://www.googleapis.com/auth/sqlservice
          sql-admin          https://www.googleapis.com/auth/sqlservice.admin
          storage-full       https://www.googleapis.com/auth/devstorage.full_control
          storage-ro         https://www.googleapis.com/auth/devstorage.read_only
          storage-rw         https://www.googleapis.com/auth/devstorage.read_write
          taskqueue          https://www.googleapis.com/auth/taskqueue
          userinfo-email     https://www.googleapis.com/auth/userinfo.email
```